CONTRIBUTING.md

```diff
@@ -55,39 +58,6 @@ If you have an idea for an enhancement to the protocol itself, please make a
 proposal by following the [NEAR Enhancement
 Proposal](https://github.com/near/NEPs/blob/master/neps/nep-0001.md) process.
 
-## Getting started
-
-- For non-technical contributions, such as e.g. content or events, see [this
-document](https://docs.nearprotocol.com/docs/contribution/contribution-overview).
-
-- For an overview of the NEAR core architecture, see [this
-playlist](https://www.youtube.com/playlist?list=PL9tzQn_TEuFV4qlts0tVgndnytFs4QSYo).
```

Comment:
Did you mean to remove this content?  It does not seem to have been moved elsewhere.

Reply:
I've actually intended to drop that in the previous PR, but got confused by the diff: 

https://github.com/near/nearcore/pull/7753/files#r985749704

The YT playlist is moved to architecture/readme, and the non-technical contributions are removed as out of scope